src/main/java/com/atlassian/jira/cloud/jenkins/deploymentinfo/client/model/DeploymentApiResponse.java

```diff
@@ -21,9 +22,10 @@ public class DeploymentApiResponse {
     @JsonCreator
     public DeploymentApiResponse(
             @JsonProperty("acceptedDeployments")
-                    final List<DeploymentKeyResponse> acceptedDeployments,
+            @Nullable final List<DeploymentKeyResponse> acceptedDeployments,
             @JsonProperty("rejectedDeployments")
-                    final List<RejectedDeploymentResponse> rejectedDeployments,
+            @Nullable final List<RejectedDeploymentResponse> rejectedDeployments,
+            @Nullable
```

Comment:
I had to make all the parameter nullable, as it seems that in some cases, the API returns null values.